amr-wind/wind_energy/ABLReadERFFcn.H

```diff
@@ -0,0 +1,17 @@
+#ifndef ABLREADERFFCN_H
```

Comment:
Can we make the name of this file and the ifdefs a bit more explicity? `ABLREADERFFUNCTION` maybe?